Library/Homebrew/extend/pathname.rb

```diff
@@ -340,8 +340,9 @@ def env_script_all_files(dst, env)
     Pathname.glob("#{self}/*") do |file|
       next if file.directory?
 
-      dst.install(file)
       new_file = dst.join(file.basename)
+      ofail "#{new_file} already exists" if new_file.exist?
```

Comment:
`raise` rather than `ofail` would be better for API like this

Reply:
OK thanks.